custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -540,10 +540,10 @@ def maybe_build_image(self, docker_image_or_directory):
         try:
             client.images.get(docker_image_or_directory)
             return docker_image_or_directory
-        except docker.errors.ImageNotFound:
+        except (docker.errors.ImageNotFound, docker.errors.APIError):
```

Comment:
Why are you adding this ?

Reply:
We should instead of doing this, check if its a directory first. IDK why I didn't do that.